### DISCUSSION TITLE
Anzeige Erzeugung/Verbrauch minimale Breite

### DISCUSSION THREAD

**Discussion by Zeppelin-PV** [solectrus/solectrus#2849] [General]
ich hätte da mal eine Vorstellung bezüglich der generellen Anzeige in Erzeugung und Verbrauch
Die Balken werden mit Abnahme der Werte im schmäler und sind ab einem gewissen Stand zwar noch da, aber es können keine Werte mehr angezeigt werden.
Wäre es nicht schön wenn eine gewisse Breite stehen bleiben könnte um die Werte (dreistellig in Watt) anzuzeigen ?
Erst bei Wert null würde ausgeblendet.
Das hätte Vorteile bezüglich kleinen Werten die lange Zeit präsent sind und dennoch interassant sind was das anzeigen anbelangt
Ist mir so eingefallen wegen der Diskussion #2621

**Comment by StevieC121176** [comment]
Das würde aber das Verhältnis der Balken zueinander in Verbindung mit den Werten verfälschen. Besser wäre dafür die Sicht "Einzelwerte"

**Reply by Zeppelin-PV** [reply]
... stimm ich Dir zu
aber ob das etwas verschobene Verhältnis wirklich eine so große Rolle spielt ?
Das Zusammenspiel einzelner Werte zueinander macht es ja aus, was interessant ist. 
hier wäre nochmals in Erinnerung zu rufen das übereinanderlegen zweier oder mehreren Diagramm Graphiken #2332
Die Kachelanzeige benutze ich kaum, bis gar nicht, weil ich dafür keine Anwendung habe; sprich Tablet, Monitor etc mit Daueranzeige (fully Kiosk Browser)

**Reply by Zeppelin-PV** [reply]
habe festgestellt dass die kleinste Breite bei 0,2kW bzw. 200W noch mit Text und kleinem Symbol angezeigt wird.
Ich fände es nicht so schlimm wenn dieser schmale Balken stehen bleibt bei Werten <200W bis Null

**Comment by ledermann** [comment]
@Zeppelin-PV Danke für deinen Vorschlag, ich schätze Vorbesserungsvorschläge immer sehr :) 

In diesem Punkt sehe ich das aber genau wie @StevieC121176 - die Höhe der einzelnen Bilanzsegmente entspricht dem relativen Anteil an der Summe. Das halte ich für ein wesentliches Element der Darstellung. Wenn wenig Platz vorhanden ist, werden Schrift und Icon kleiner und fallen schließlich ganz weg. Das habe ich sehr bewusst so implementiert.

Inhaltlich finde ich das genau richtig: Wenn ein Messwert nur wenige Prozent ausmacht, muss ich den nicht genau wissen. Grundsätzlich verfolge ich bei SOLECTRUS auch das Ziel, unwichtige Details möglichst wegzulassen. 

Per Tooltip ist der Wert auch bei kleinen Bilanzsegmenten noch erreichbar - auch wenn es da noch einen ärgerlichen kleinen Fehler mit der Aktualisierung gibt (https://github.com/solectrus/solectrus/issues/1886), den ich mir noch vornehmen muss.

**Comment by Zeppelin-PV** [comment]
... hab nochmal drüber geschlafen 
Ja, ihr habt recht - dumme Idee
... ganz schnell schließen
